quiz.js

```diff
@@ -0,0 +1,58 @@
+// adding variables to elements obtained from the Dom
+
+var height = document.getElementById("height");
+var character = document.getElementById("char");
+var button = document.getElementById("but");
+// adding event listener to the button calling to run the first function
+button.addEventListener("click", checkTree);
+
+// this function checks to make sure that values are entered
+// used ! to signify "not" and && for "and" to open prompt box if there is no height or character value
+function checkTree(){
+  if (!height.value && ! character.value){
+    alert("please enter values in both boxes");
+    // if height does not have a value
+  }   else if (!height.value){
+    alert("enter height");
+    // if there is no character value
+  }   else if (!character.value){
+    alert("enter character");
+  }   else {
+    // stating the value of the tree variable
+  var tree = {
+    height: height.value,
+    character: character.value
+  }
+  // calls the function printTree
+  printTree(tree);
+  }
+}
+// adding the even listen to the enter key to run checkTree as defined above if text is entered in the active element
+addEventListener("keydown", function(event){
+  if(event.keyCode === 13 && (height === document.activeElement || character === document.activeElement))
```

Comment:
I see what you did here, but I think `if (event.keyCode === 13)` should be sufficient